lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -459,6 +459,13 @@ inventory:
     config_get: "show inventory"
     config_get_token: ["TABLE_inv", "ROW_inv", 0, "vendorid"]
 
+ntp_config:
+  config:
+    config_get: "show running-config | include 'ntp source-interface'"
```

Comment:
Ok, I see that now.  No need to anchor then if using the show command that @chrisvanheuveln  suggested.